src/lib/molecules/ExtendedInput/index.js

```diff
@@ -89,6 +90,10 @@ const ExtendedInput = forwardRef((props, ref) => {
     // non strict equality is needed for covering 'undefined' case also
     const inputValue = isControlled ? (value != null ? value : '') : localValue;
 
+    const [isTextTruncated, setIsTextTruncated] = useState(false);
+    const ellipsisDetector = useEllipsisDetection(inputRef, [inputValue]);
```

Comment:
The variable name is confusing it duplicates the hook name which is not intuitive what I would expect from a hook, it can be renamed e.g. `isTruncated` as the hook returns a `boolean` value or something like `isTextEllipsis`,